### PR TITLE
ay 7aga

### PR DESCRIPTION
**Pull request by khaled-dev**
small change

### CODE REVIEW

**Review by mostafanabil198 — APPROVED** [review]

**Comment by ahmednabilespace** [comment]
7lw gedan

File: file-tani-5ales [hunk 1]
```diff
@@ -1 +1,9 @@
 yo
+ldfkdjf
+dkfjdkjf
+dfs
+ds
+f
```
Comment (khaled-dev) [inline file-tani-5ales:6]: kindly make it capital

File: file-tani-5ales [hunk 2]
```diff
@@ -1 +1,9 @@
...
+sdf
+ds
+
```
Comment (khaled-dev) [inline file-tani-5ales:9]: please remove this extra space

**Review by amohamed97 — APPROVED** [review]

File: file-tani-5ales (hunk 1 above)
Comment (ahmednabilespace) [reply file-tani-5ales:6]: OK

**Comment by ahmed-aliraqi** [comment]
Nice work Mr. BolBol @khaled-dev

**Comment by HatemElsaid** [comment]
> Nice work Mr. BolBol @khaled-dev

Good Help

File: file-tani-5ales (hunk 2 above)
Comment (HatemElsaid) [reply file-tani-5ales:9]: Yes
To Be Clean Commit ,,

**Pull request merged**